### PR TITLE
AC_Fence: clarify bit 2 in FENCE_ENABLE parameter

### PR DESCRIPTION
**Pull request by peterbarker**
these aren't just polygon fences ('though they used to be)

### CODE REVIEW

File: libraries/AC_Fence/AC_Fence.cpp [hunk 1]
```diff
@@ -52,8 +52,8 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @Param: TYPE
     // @DisplayName: Fence Type
     // @Description: Enabled fence types held as bitmask
-    // @Bitmask{Rover}: 1:Circle,2:Polygon
-    // @Bitmask{Copter, Plane, Sub}: 0:Max altitude,1:Circle,2:Polygon,3:Min altitude
+    // @Bitmask{Rover}: 1:Circle,2:Incusion/Exclusion Circles+Polygons
```
Comment (rmackay9) [inline libraries/AC_Fence/AC_Fence.cpp:55]: Little typo in "inclusion", missing the "l".

**Comment by rmackay9** [comment]
Maybe we should rename "1" as well to "Circle around Home" to clarify vs "2"'s circles.

File: libraries/AC_Fence/AC_Fence.cpp (hunk 1 above)
Comment (peterbarker) [reply libraries/AC_Fence/AC_Fence.cpp]: Fixed.

**Comment by peterbarker** [comment]
> Maybe we should rename "1" as well to "Circle around Home" to clarify vs "2"'s circles.

I've made a change here, but went with "Circle Centered on Home", rather than "Circle around Home", which could be confused with an action rather than a fence shape.

**Review by tridge — APPROVED** [review]

**Review by rmackay9 — APPROVED** [review]

**Pull request merged**